### PR TITLE
Fixed a few typos and updated version details.

### PR DESCRIPTION
**Pull request by jeana-redhat**
@vikram-redhat 
These changes are in the **Contribute to OpenShift documentation** pages.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
LGTM

**Comment by jeana-redhat** [comment]
@openshift/team-documentation Ready for y'all

**Comment by bergerhoffer** [comment]
@vikram-redhat If it's not already set up, can you set up the preview-bot for @jeana-redhat please?

File: contributing_to_docs/contributing.adoc [hunk 1]
```diff
@@ -101,15 +101,15 @@ This first line is unconditionalized, and will appear for all versions.
 
 \ifdef::openshift-online[]
 This line will only appear for OpenShift Online.
-endif::[]
+\endif::[]
 
 \ifdef::openshift-enterprise
```
Comment (bergerhoffer) [inline contributing_to_docs/contributing.adoc]: So the endifs are now showing up, but there are still slashes rendering before the second and third ifdefs. Suggest trying to play with this a little more to see if you can get rid of that as well.

**Review by bergerhoffer — COMMENTED** [review]
Edits look good, just one suggestion to see if you can get rid of the slashes in front of the ifdefs. If not, no worries and we can merge these in.

**Comment by bergerhoffer** [comment]
Thanks for the update, looks great! Merging

**Pull request merged**